src/test/java/com/amazon/ion/conformance/expectations.kt

```diff
@@ -141,20 +142,20 @@ private fun TestCaseSupport.denotesModelContent(modelContent: AnyElement, reader
             assertEquals(IonType.STRING, reader.type, failureContext)
             assertEquals(modelContent.stringValue, reader.stringValue(), failureContext)
         }
-        is SexpElement -> when (modelContent.head) {
+        is SeqElement -> when (modelContent.head) {
```

Comment:
🗺️ Switch to `SeqElement` ensures that the `denotes` clause works even if the test is written in JSON instead of Ion. That might not be very relevant in ion-java, but it was an easy change to get some more completeness.